src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -200,6 +201,11 @@ protected override void EnableService()
                 return;
             }
 
+            _workQueue = new AsyncBatchingWorkQueue<(bool solutionChanged, ProjectId? changedProject)>(
+                TimeSpan.FromSeconds(1),
+                ProcessWorkQueueAsync,
+                this.DisposalToken);
```

Comment:
❕ This is missing the `IAsynchronousOperationListener` which is used to expedite processing

Reply:
indeed.  good catch.